quarkus/admin/src/main/java/org/apache/polaris/admintool/BootstrapCommand.java

```diff
@@ -46,8 +46,6 @@ public class BootstrapCommand extends BaseCommand {
 
   @Override
   public Integer call() {
-    warnOnInMemory();
```

Comment:
Any reason not to leave this in?

Reply:
We are removing in-memory support completely. This tool doesn't make any sense with in-memory metastores.

Reply:
Oh, I see. I was wondering what would happen if someone explicitly re-configured it back to the old implementation. But it wouldn't make sense to do that anyway, so it doesn't matter. Removing dead code sounds good to me.